federated_learning/nvflare/build_docker_provision.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+DOCKER_IMAGE=projectmonai/monaiwfl:0.1
```

Comment:
what's this docker mean? "MONAI with FL"? Maybe: "monai_nvflare"?

Reply:
Looks good, I'll change the name here.